README.md

```diff
@@ -47,6 +47,7 @@ Swarm Control Planes.
   - v9.1
     - [v9.1.0](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.1.0.md)
   - v9.0
+    - [v9.0.6](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.0.6.md)
```

Comment:
@tfussell @MarcelMue FYI I've added this link to the release notes

Reply:
Thanks. The "separate releases" PR adds a CI check for this since it's easy to forget.